src/Controller/ErrorController.php

```diff
@@ -57,7 +57,7 @@ public function beforeRender(Event $event) : void
         parent::beforeRender($event);
 
         $this->viewBuilder()->setClassName('App\View\AppView');
-        $this->viewBuilder()->setTemplatePath('Error');
+        $this->viewBuilder()->setTemplatePath('Pages/Error');
```

Comment:
(see comment above)

Since `ErrorController extends AppController`, if the advice in `AppController` is followed this becomes unnecessary.